Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -168,12 +168,6 @@ public override string GetDriverConfigsEditPageName(Agent.eDriverType driverSubT
         public string Proxy { get; set; }
```

Comment:
please move all common properties between selenium and playwright to GingerWebDriver.cs